docs/UserGuide.md

```diff
@@ -260,6 +329,11 @@ _{give a remark to the recipe}_
 
 --------------------------------------------------------------------------------------------------------------------
 
+# Glossary
```

Comment:
Can include the explanation of consumption also.

Reply:
Good idea ok